devices/stm32g473.yaml

```diff
@@ -18,3 +18,5 @@ _include:
  - ../peripherals/opamp/opamp_g4_common.yaml 
  - ../peripherals/opamp/opamp_g4_opamp4_5.yaml 
  - ../peripherals/opamp/opamp_g4_opamp6.yaml 
+ - ./common_patches/rename_USB_FS_peripheral_to_USB.yaml
+ - ./common_patches/g4_usb.yaml
```

Comment:
I believe these are probably needed for other G4 series as well but I do not have a good way to test those changes. Should they be added?